share/governance.md

```diff
@@ -200,10 +234,12 @@ to its success. To that end, decision making must be done in a transparent,
 open fashion. No decisions about the project’s direction, bug fixes or features
 may be done without community involvement and participation. Discussions must
 begin at the earliest possible point on a topic; the community’s participation
-is vital during the entire decision-making process.
+is vital during the entire decision-making process. While conversations happen
+on many mediums, decisions must be made through the mailing list.
 
 ## Election
 
 The details of the election process are up to the elections officer, subject to
 approval by the PMC. The users, collaborators, and contributors all get to vote
-in the election.
+in the election. The election will pick members of the PMC as well as the
```

Comment:
People could indicate the subcommittees they are interested in for the election.

Reply:
It isn't clear here that we're talking about multiple elections., pretty much throughout  

Are we looking at each PMC role being assigned to a subcommittee (so you apply to a specific PMC role?)  Or should we just get the best PMC we can, and then they can work out who is going to go to which subs.  (There shouldn't be any reason why the entire PMC can't show up to every sub meeting.

For subcommittee placement, is there any reason to vote on those, or should it just be an open meeting that folks can come to?

Reply:
I did not mean to give the impression of multiple elections. I thought one election and you can specify the different committees you are interested in. If PMC stays at 5, not everyone will want to be in PMC and some may just want to be in certain subcommittees.